packages/browser/qa/__tests__/destinations.test.ts

```diff
@@ -61,7 +68,7 @@ describe('Destination Tests', () => {
           host: url.host + url.pathname,
           queryParams: Array.from(url.searchParams.entries())
             .map((entry) => entry[0])
-            .filter((p) => !p.match(/^\d+$/))
```

Comment:
Why this change? It looks like it was filtering out any query string parameters where the key was a number which...I don't actually know why we do that (I don't think a.js does anything with query string parameters that are just numbers though...), but I don't think we want the opposite of that. It also doesn't match what we do for analytics-next requests so those won't be matching now.

Reply:
Oops, this was an accident from when I was fixing the types (and testing them). Hmm, weird that it didn't trigger a test failure =)